readme.md

```diff
@@ -29,3 +29,25 @@ namespace Sample
 
 - Generates UTF-8 binary data from string literals (UTF-16).
 - The UTF-8 data is optimized to avoid allocation. see: [C# ReadOnlySpan and static data](https://vcsjones.dev/2019/02/01/csharp-readonly-span-bytes-static/)
+
+## NuGet
+
+[![NuGet](https://img.shields.io/nuget/v/StringLiteralGenerator?style=flat-square)](https://www.nuget.org/packages/StringLiteralGenerator)
+
+You may also need to add a reference to `Microsoft.Net.Compilers.Toolset`.
```

Comment:
Is `Microsoft.Net.Compilers.Toolset` needed even when using .NET 5 SDK RC1 or later?

Reply:
I tried these:

| SDK | needed |
| --- | --- |
| RC 2 | no |
| RC 1 | didn't test |
| previews | yes |

That's why I've written `may`. 
Is it better to explicitely call this out, like
```suggestion
For versions earlier than .NET 5 SDK RC2 you may also need to add a reference to `Microsoft.Net.Compilers.Toolset`.
```
?